.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: node_js
+
+node_js:
+
+  - "12.18.3"
+
+before_script:
+
+  - npm install
+
+script:
+
+  - "npm test"
+
+after_script:
+
+  - "npm run coverage"
+
+notifications:
+
+  email: false
+
+sudo: false
```

Comment:
you may add new line at the end of file to resolve eslint errors

Reply:
Yes, Right I have worked on it ! Thanks @Rwandarushya